tests/conftest.py

```diff
@@ -87,8 +87,14 @@ def mock_download(self, url, dest):
     if dest.exists():
         os.unlink(str(dest))
 
-    if PY2 and WINDOWS:
-        shutil.copyfile(str(fixture), str(dest))
+    if WINDOWS:
```

Comment:
I'm tempted to replace this whole block with:
```python
	    try:
                os.symlink(str(fixture), str(dest))
            except OSError:
                # symlink doesn't work on windows in python 2 or without admin permissions
                shutil.copyfile(str(fixture), str(dest))

Reply:
So os.symlink isn't available on Windows in Python 2 which would then maybe throw an `AttributeError` instead? I was thinking along the same lines as your try/except and also was unsure given the previous use of flags :)

Reply:
Makes sense. Perhaps an inline comment is in order regarding the reasoning (admin privileges etc)?

Reply:
Added and pushed